### ISSUE TITLE
Enhancement Request - add multiple emails 

### ISSUE THREAD

**Issue by CMaxim77** [moralmunky/Home-Assistant-Mail-And-Packages#79]
Is there anyway to be able to add multiple email addresses per delivery accounts?

I use different email addresses for diffrrrnt carriers, specifically Amazon.

Is there a way to set this up maybe even multiple times in HA so if we do use a different email we can still get sensors for all of the included services?

Thanks for all the work you put into this for us!

**Comment by firstof9** [comment]
We'll see what we can do.

**Comment by firstof9** [comment]
Give these files a shot if you'd like to test:

https://raw.githubusercontent.com/firstof9/Home-Assistant-Mail-And-Packages/config-flow-v3/custom_components/mail_and_packages/config_flow.py
https://raw.githubusercontent.com/firstof9/Home-Assistant-Mail-And-Packages/config-flow-v3/custom_components/mail_and_packages/sensor.py
https://raw.githubusercontent.com/firstof9/Home-Assistant-Mail-And-Packages/config-flow-v3/custom_components/mail_and_packages/const.py

**Comment by CMaxim77** [comment]
Thank you! I really appreciate you

I will let you know how the testing goes!

**Comment by firstof9** [comment]
Keep in mind you will have doubles of the sensors, likely appended with _2 for the 2nd account. I added an attribute to the sensors to display the mail server they belong to as well to help.

**Comment by CMaxim77** [comment]
Sorry I haven’t had a chance to report back sooner. Work has been crazy lately. I was able to set a second instance of this, but wasn’t able to get amazon to work. When I set up the second account the working original one stops. Weirdly it don’t append the sensors to _2 so maybe that’s why. I’m going to try to setup again over the weekend And re send the new files just to be sure I did everything correctly. I just wanted you to know. Thanks for your efforts on this!
